### PR TITLE
Fix #2147: Rtl support for TopicPractice tab

### LINKED ISSUE #2147: [RTL] High fi TopicPractice tab.
`Topic-Practice` needs RTL support as shown in below screen.
TP (Practice, Unchecked)

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by FareesHussain**
## Explanation

Fixes #2147 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

Mobile Portrait | Mobile Landscape 
--- | --- 
 | 

Tablet Portrait | Tablet Landscape 
--- | --- 
 |

### CODE REVIEW

**Comment by FareesHussain** [comment]
@rt4914 please check

**Comment by rt4914** [comment]
> @rt4914 please check

Hello @FareesHussain PTAL https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section

**Review by rt4914 — CHANGES_REQUESTED** [review]
@FareesHussain 
Why is `tools:ignore="RtlCompat"` needed? Because if I am removing this, it does not affect UI. Am I missing something?

**Comment by FareesHussain** [comment]
> @FareesHussain
> Why is `tools:ignore="RtlCompat"` needed? Because if I am removing this, it does not affect UI. Am I missing something?

i added this cause it shows a error with a suggestion to add it when I  add `android:textAlignment="viewStart"` in a checkbox
but it works even without it

**Comment by rt4914** [comment]
> > @FareesHussain
> > Why is `tools:ignore="RtlCompat"` needed? Because if I am removing this, it does not affect UI. Am I missing something?
> 
> i added this cause it shows a error with a suggestion to add it when I add `android:textAlignment="viewStart"` in a checkbox
> but it works even without it

@FareesHussain Okay, in that case you can fix this by using `android:gravity="start"`

**Comment by FareesHussain** [comment]
@rt4914 `android:gravity="start"` is not working for check box

**Comment by rt4914** [comment]
> @rt4914 `android:gravity="start"` is not working for check box

@FareesHussain Actually i did check that code and it was working. Can you please mention what error you are getting?

**Comment by FareesHussain** [comment]
I'm sorry I was little confused i'll push those changes

File: app/src/main/res/layout-sw600dp-land/topic_practice_subtopic.xml [hunk 1]
```diff
@@ -42,15 +42,16 @@
...
         android:minHeight="22dp"
         android:text="@{viewModel.subtopic.title}"
+        android:textAlignment="viewStart"
         android:textColor="@color/oppiaPrimaryText"
         android:textSize="16sp"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toTopOf="parent"/>
```
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/topic_practice_subtopic.xml]: Please add space before closing tag.
`app:layout_constraintTop_toTopOf="parent" />`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@FareesHussain Just one nit change suggested, rest all looks good. Thanks.

File: app/src/main/res/layout-sw600dp-land/topic_practice_subtopic.xml (hunk 1 above)
Comment (FareesHussain) [reply app/src/main/res/layout-sw600dp-land/topic_practice_subtopic.xml]: Done please check

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**